### PR TITLE
Added RoboSub post

### PR DESCRIPTION
**Pull request by jspspike**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -13,6 +13,8 @@
                 <a class="dropdown-item" href="{{ site.baseurl }}/about.html">Everything</a>
                 <div class="dropdown-divider"></div>
                 <a class="dropdown-item" href="{{ site.baseurl }}/about/robotathon.html">Robotathon</a>
+                <a class="dropdown-item" href="{{ site.baseurl }}/about/robosub.html">RoboSub</a>
+                <a class="dropdown-item" href="{{ site.baseurl }}/about/curm.html">CURM</a>
```
Comment (rrbutani) [inline _includes/header.html:17]: @jspspike I think this was a mistake

Comment (jspspike) [reply _includes/header.html:17]: Oops, fixed now

**Review by rrbutani — APPROVED** [review]

**Review by RosannaR — CHANGES_REQUESTED** [review]
There is an issue with line 16.

**Comment by jspspike** [comment]
> There is an issue with line 16.

Removed that line

**Review by RosannaR — APPROVED** [review]

**Pull request merged**